packages/reserve-site/README.md

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Nit: was the extra spacing in this file intentional? Consider installing https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint